Package@swift-5.5.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.4
+// swift-tools-version:5.5
```

Comment:
I don't think we need this change since we are not using any features of the new tools version in here